2.evaluate_models/lr_plate_3_3p_cp_fs_data/evaluate_model.ipynb

```diff
@@ -0,0 +1,401 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "jukit_cell_id": "AAuDgo76ka"
+      },
+      "source": [
+        "# Determine the best features using a Logistic Regression Model"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "jukit_cell_id": "SyZ3qa8iz3"
+      },
+      "source": [
+        "## Imports"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "jukit_cell_id": "FKd1dIo45E"
+      },
+      "source": [
+        "import sys\n",
+        "import pandas as pd\n",
+        "import numpy as np\n",
+        "from pathlib import Path\n",
+        "from sklearn.preprocessing import LabelEncoder\n",
+        "\n",
+        "from sklearn.metrics import (\n",
+        "    confusion_matrix,\n",
+        "    ConfusionMatrixDisplay,\n",
+        "    precision_score,\n",
+        "    accuracy_score,\n",
+        "    recall_score,\n",
+        "    f1_score,\n",
+        ")\n",
+        "from sklearn.model_selection import train_test_split\n",
+        "\n",
+        "import matplotlib.pyplot as plt\n",
+        "import seaborn as sns\n",
+        "\n",
+        "from joblib import dump, load"
+      ],
+      "outputs": [],
```

Comment:
This notebook looks like it might have had the outputs cleared (along with others in this PR). Would it make sense to keep the output within the notebook to help demonstrate the findings?

Reply:
That is a good point, the way I have been developing, I normally run code in a specialized python file, and then covert those files to a jupyter notebook and a normally formatted python file. I think I will try rerunning the code in my jupyter notebook from now on with nbconvert.